server/channels/app/platform/web_hub.go

```diff
@@ -497,6 +505,8 @@ func (h *Hub) Start() {
 						return
 					}
 					if webConn.ShouldSendEvent(msg) {
+						msg := h.runBroadcastHooks(msg, webConn)
```

Comment:
Broadcast hooks are the last thing that happens before we send the event from the hub to an individual connection so that they'll be called for each event when something is broadcast to a whole channel or team